src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -11007,7 +11007,13 @@ def infer_sorted_column_labels(
         # Inherit the index names from the dataframe.
         if squeeze_self:
             # self is a Series, other a DataFrame.
-            series = self.to_pandas().squeeze()
+            series_self = self.to_pandas()
+            if series_self.size > 1:
```

Comment:
nit: can shorten to `series = series_self.squeeze() if series_self.size > 1 else series_self.squeeze(axis=0)`